fmt.sh

```diff
@@ -133,6 +140,7 @@ run dade-copyright-headers "$dade_copyright_arg" .
 if [ "$diff_mode" = "true" ]; then
   check_diff $merge_base '\.hs$' hlint -j4
   check_diff $merge_base '\.java$' javafmt "${javafmt_args[@]:-}"
+  check_diff $merge_base '\.(ts|tsx)$' npx pprettier --check
```

Comment:
Is the name `pprettier` a typo? I see it appearing again, so not sure. Furthermore, shouldn't you use `prettier_args` here?